src/platform/packages/shared/response-ops/alerts-delete/components/modal.tsx

```diff
@@ -59,6 +63,17 @@ const getThresholdInDays = (threshold: number, thresholdUnit: EuiSelectOption) =
   }
 };
 
+const dateTimeFormat = (locale: string) => {
```

Comment:
Shouldn't this come from `useUiSetting<string>(UI_SETTINGS.DATE_FORMAT);`?

Reply:
Oh I didn't realize that, but still, the date format should normally come from the kibana user settings.

Reply:
I was wrong in assuming that using `moment` would break CI. It worked! https://github.com/elastic/kibana/pull/219544/commits/5ed25d578abb0771c9c366a128262276b7916ac9